apache_exporter.go

```diff
@@ -58,17 +59,22 @@ func NewExporter(uri string) *Exporter {
 		}),
 		accessesTotal: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "accesses_total"),
-			"Current total apache accesses",
+			"Current total apache accesses (*)",
```

Comment:
Better not include these.
The installation details are available from the readme.

Its not referring to anything, there's no notes with the output consumed by prometheus.